{"traceId":"5ae5ea54bb9d0363c13e9440ab8e83a2","displayName":"menuSuggestionFlow","startTime":1761146671427,"endTime":1761146671431.9272,"spans":{"2072fafe50a78a27":{"spanId":"2072fafe50a78a27","traceId":"5ae5ea54bb9d0363c13e9440ab8e83a2","startTime":1761146671427,"endTime":1761146671431.9272,"attributes":{"genkit:type":"action","genkit:metadata:subtype":"flow","genkit:name":"menuSuggestionFlow","genkit:isRoot":true,"genkit:path":"/{menuSuggestionFlow,t:flow}","genkit:input":"\"seafood\"","genkit:state":"error","genkit:isFailureSource":true},"displayName":"menuSuggestionFlow","links":[],"instrumentationLibrary":{"name":"genkit-tracer","version":"v1"},"spanKind":"INTERNAL","sameProcessAsParentSpan":{"value":true},"status":{"code":2,"message":"INVALID_ARGUMENT: Must supply a `model` to `generate()` calls."},"timeEvents":{"timeEvent":[{"time":1761146671431.6162,"annotation":{"attributes":{"exception.type":"400","exception.message":"INVALID_ARGUMENT: Must supply a `model` to `generate()` calls.","exception.stacktrace":"GenkitError: INVALID_ARGUMENT: Must supply a `model` to `generate()` calls.\n    at resolveModel (C:\\Users\\boshr\\StudioProjects\\xenodate\\functions\\node_modules\\@genkit-ai\\ai\\src\\model.ts:511:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async toGenerateActionOptions (C:\\Users\\boshr\\StudioProjects\\xenodate\\functions\\node_modules\\@genkit-ai\\ai\\src\\generate.ts:497:25)\n    at async generate (C:\\Users\\boshr\\StudioProjects\\xenodate\\functions\\node_modules\\@genkit-ai\\ai\\src\\generate.ts:391:18)"},"description":"exception"}}]}}}}